docs/Manual/Deployment/Kubernetes/Drain.md

```diff
@@ -0,0 +1,444 @@
+# Draining Kubernetes nodes
+
+{% hint 'danger' %}
+If Kubernetes nodes with ArangoDB pods on them are drained carelessly
+data loss can occur! The proper procedure is described below.
+{% endhint %}
+
+For maintenance work in k8s it is sometimes necessary to drain a k8s node,
+which means removing all pods from it. Kubernetes offers a standard API
+for this and our operator supports this - to the best of its ability.
+
+Draining nodes is easy enough for stateless services, which can simply be
+re-launched on any other node. However, for a stateful service this
+operation is more difficult, and as a consequence more costly and there
+are certain risks involved, if the operation is not done carefully
+enough. To put it simply, the operator must first move all the data
+stored on the node (which could be in a locally attached disk) to
+another machine, before it can shut down the pod gracefully. Moving data
+takes time, and even after the move, the distributed system ArangoDB has
+to recover from this change, for example by ensuring data synchronicity
+between the replicas in their new location.
+
+{% hint 'warning' %}
+Therefore, a systematic drain of all k8s nodes in sequence has to follow
+a careful procedure, in particular to ensure that ArangoDB is ready to
+move to the next step. This is necessary to avoid catastrophic data
+loss, and is simply the price one pays for running a stateful service.
+{% endhint %}
```

Comment:
@sleto-it I added a red hint box at the top, so maybe we can remove your warning here?